src/EPR.RegulatorService.Frontend.Core/Models/RegistrationSubmissions/RegulatorDecisionRequest.cs

```diff
@@ -0,0 +1,14 @@
+namespace EPR.RegulatorService.Frontend.Core.Models.RegistrationSubmissions;
+
+using EPR.RegulatorService.Frontend.Core.Enums;
+
+public class RegulatorDecisionRequest
```

Comment:
Can you rename this to RegulatorRegistrationDecisionRequest please?

Reply:
yes, sure I am working on these. Thanks for the name

Reply:
done, as discussed with @JulianYoungEviden